commit dd0859dccbe291cf8179a96390f5c0e45cb9af1d
Author: James Morris <jmorris@namei.org>
Date:   Wed Feb 15 00:17:24 2017 +1100

    security: introduce CONFIG_SECURITY_WRITABLE_HOOKS
    
    Subsequent patches will add RO hardening to LSM hooks, however, SELinux
    still needs to be able to perform runtime disablement after init to handle
    architectures where init-time disablement via boot parameters is not feasible.
    
    Introduce a new kernel configuration parameter CONFIG_SECURITY_WRITABLE_HOOKS,
    and a helper macro __lsm_ro_after_init, to handle this case.
    
    Signed-off-by: James Morris <james.l.morris@oracle.com>
    Acked-by: Stephen Smalley <sds@tycho.nsa.gov>
    Acked-by: Casey Schaufler <casey@schaufler-ca.com>
    Acked-by: Kees Cook <keescook@chromium.org>

diff --git a/security/Kconfig b/security/Kconfig
index d900f47eaa68..3ff1bf91080e 100644
--- a/security/Kconfig
+++ b/security/Kconfig
@@ -31,6 +31,11 @@ config SECURITY
 
 	  If you are unsure how to answer this question, answer N.
 
+config SECURITY_WRITABLE_HOOKS
+	depends on SECURITY
+	bool
+	default n
+
 config SECURITYFS
 	bool "Enable the securityfs filesystem"
 	help